packages/breadcrumbs/src/styled/StyledBreadcrumb.ts

```diff
@@ -6,19 +6,21 @@
  */
 
 import styled from 'styled-components';
-import classNames from 'classnames';
-import BreadcrumbStyles from '@zendeskgarden/css-breadcrumbs';
-import { retrieveComponentStyles, isRtl } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'breadcrumbs.list';
 
-export const StyledBreadcrumb = styled.ol.attrs(props => ({
-  'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION,
-  className: classNames(BreadcrumbStyles['c-breadcrumb'], {
-    // RTL
-    [BreadcrumbStyles['is-rtl']]: isRtl(props)
-  })
-}))`
-  ${props => retrieveComponentStyles(COMPONENT_ID, props)};
+export const StyledBreadcrumb = styled.ol.attrs(() => {
+  return {
+    'data-garden-id': COMPONENT_ID,
+    'data-garden-version': PACKAGE_VERSION
+  };
+})`
+  ${props => `
+    display: flex;
+    margin: 0;
+    padding: 0;
```

Comment:
These are CSS resets that have useful comments on why they exists in their respective CSS files. Those comments should be brought over during the CSS-in-JS migration as they will be helpful for understanding why these exists.